public/js/core.js

```diff
@@ -1,21 +1,17 @@
 var tarchive = angular.module('tarchive', ["APIService", "ngTable", "ngSanitize"]);
 
-tarchive.controller('CoreController', ['$scope', '$filter', 'API', 'NgTableParams', function($scope, $filter, API, NgTableParams) {
+tarchive.config(['$compileProvider', function( $compileProvider ) {
+    $compileProvider.aHrefSanitizationWhitelist(/^\s*(https?|http|ftp|mailto|data|chrome-extension):/);
```

Comment:
I don't think anything other than the `data` is necessary since thats the only kind of url we are using